drivers/timer/Kconfig.x86

```diff
@@ -5,6 +5,7 @@
 
 choice
 	prompt "Default System Timer"
+	default HPET_TIMER if SOC_FAMILY_INTEL_ISH || SOC_IA32 || SOC_LAKEMONT
```

Comment:
... hence I think HPET should not be marked deprecated.

Reply:
Removed deprecation of HPET